tests/ui/test/specs/view-locations.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { test, expect } from "@playwright/test";
+import { allure } from "allure-playwright";
+import ViewLocationsPage from "../../src/pages/view-locations-page";
+import BasePage from "../../src/pages/base-page";
+
+let viewLcPage: ViewLocationsPage;
+let basePage: BasePage;
+let pageTitle: string = "UEC Capacity Management";
+
+test.describe("As a user I want to be able to view the locations", () => {
+  test.beforeEach(async ({ page }, testInfo) => {
+    await allure.parentSuite(testInfo.project.name);
+    await allure.suite("Tests for view locations journeys");
+    await allure.subSuite("Tests for locations landing page");
+    viewLcPage = new ViewLocationsPage(page);
+    basePage = new BasePage(page);
+    await test.step("Navigate to landing page", async () => {
+      await page.goto("/prototype");
```

Comment:
The prototype will be moving to another repo so the "/prototype" url along with this login page will be removed from this repo until it is officially built. 
This will impact these tests and cause them to fail and a review might be needed on if we should be adding tests directly for the prototype code.

Reply:
When the repo is changed - The test we writing going to fail, All the tests get reviewed again during the sprint.  We can update the url in Env. file and any locater changes during the sprint testing. 
This is more test driven approach